src/main/scala/KindProjector.scala

```diff
@@ -200,19 +200,19 @@ class KindRewriter(plugin: Plugin, val global: Global)
 
       tree match {
         case Template(parents, self, body) =>
-          Template(parents.map(transform), self, body.map(transform))
+          treeCopy.Template(tree, parents.map(transform), self, body.map(transform))
```

Comment:
Oh! I remember.

I think there is (or was) a bug where the `parents` would not be transformed normally.